torch/_library/triton.py

```diff
@@ -40,7 +40,7 @@ def triton_op(
 
     Note that ``fn`` must only consist of calls to PyTorch-understood
     operators and triton kernels. Any triton kernels called inside ``fn``
-    must be wrapped in a call to :func:`torch._library.wrap_triton``.
+    must be wrapped in a call to :func:`torch._library.wrap_triton`.
```

Comment:
This is awkward, the API name should be torch.library.wrap_triton. Could you change that too?